src/PopEngine/index.ts

```diff
@@ -146,6 +146,13 @@ export class PopEngine {
         document.body.appendChild(container);
       }
 
+      if (this._isSafari()) {
```

Comment:
```
 +      if (this._isSafari()) {
 +        document.querySelector('div[pop-id="' + groupId + '"]').addEventListener('transitionend', function(): void {
 +        console.log('transition!!');
 +        container.classList.remove('hidden');
 +      }.bind(this), true);
```

When a pop is reused, is another handler created on the same pop? I'm thinking this code needs to remove itself too once transitionend triggers.